src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,877 @@
+				selectNewCDP();
+			}
+		});
+		selectCDPButton.setToolTipText("Select a different CDP");
+		selectCDPButton.setBounds(541, 56, 39, 23);
+		controlsPane.add(selectCDPButton);
+
+		JPanel LogPanel = new JPanel();
+		LogPanel.setSize(new Dimension(400, 400));
+		LogPanel.setMinimumSize(new Dimension(400, 400));
+
+		LogPanel.setLayout(new BorderLayout(0, 0));
+
+		textArea = new JTextArea();
+		textArea.setFont(new Font("Courier New", Font.PLAIN, 13));
+		textArea.setSize(new Dimension(360, 360));
+		JScrollPane scrollPane = new JScrollPane(textArea);
+		LogPanel.add(scrollPane, BorderLayout.CENTER);
+
+		JPanel panel = new JPanel();
+		panel.setPreferredSize(new Dimension(50, 50));
+		FlowLayout flowLayout_1 = (FlowLayout) panel.getLayout();
+		flowLayout_1.setHgap(45);
+		LogPanel.add(panel, BorderLayout.SOUTH);
+
+		JButton clearTextAButton = new JButton("Clear");
+		clearTextAButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				clearTextArea();
+			}
+
+		});
+		panel.add(clearTextAButton);
+
+		JButton copyButton = new JButton("Copy");
+		copyButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				ClipboardUtilities.put(textArea.getText());
+			}
+		});
+		copyButton.setToolTipText("Copy text into clipboard");
+		panel.add(copyButton);
+
+		JPanel bottomPane = new JPanel();
+		FlowLayout flowLayout = (FlowLayout) bottomPane.getLayout();
+		flowLayout.setHgap(125);
+		add(bottomPane, BorderLayout.SOUTH);
+
+		JButton submitRunButton = new JButton("Submit");
+		submitRunButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+			    submitSelectedJudgesSolutions();
+			}
+		});
+
+		JButton resetButton = new JButton("Reset");
+		resetButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				resetFields(true);
+			}
+		});
+		resetButton.setToolTipText("Reset back to default settings");
+		bottomPane.add(resetButton);
+		submitRunButton.setToolTipText("Submit selected sample runs");
+		bottomPane.add(submitRunButton);
+
+//		centerPane2.add(controlsPane, BorderLayout.CENTER);
+		JScrollPane leftSide = new JScrollPane(controlsPane);
+		
+		JPanel submitNoSamplesPane = new JPanel();
+		submitNoSamplesPane.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+		FlowLayout fl_submitNoSamplesPane = (FlowLayout) submitNoSamplesPane.getLayout();
+		fl_submitNoSamplesPane.setAlignment(FlowLayout.LEFT);
+		submitNoSamplesPane.setBounds(33, 131, 650, 32);
+		controlsPane.add(submitNoSamplesPane);
+
+		checkBoxSubmitFailingSamples = new JCheckBox("Submit Failing Samples");
+		checkBoxSubmitFailingSamples.setVerticalAlignment(SwingConstants.TOP);
+		submitNoSamplesPane.add(checkBoxSubmitFailingSamples);
+		checkBoxSubmitFailingSamples.setSelected(true);
+		checkBoxSubmitFailingSamples.setToolTipText("Submt all non-AC (Yes) submissions");
```

Comment:
Please correct the typo in this tooltip  ("Submt" should be "Submit").

Also, I think the tooltip should make it clear that it is the JUDGE'S submissions that are being "submitted".  As worded, it could be confused with team submissions.  I think a better wording for the tooltip would be "Submit all non-AC (non-Yes) judge's solutions".